<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>withdrawal</class>
 <widget class="QDialog" name="withdrawal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn40">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFetchTransactions">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>80</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDifSum">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>150</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelChosenSum">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Valittu summa = </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>340</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn100">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>410</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 255, 10);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn20">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>390</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn8">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>440</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>440</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnArrow8">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>290</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn0">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>490</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>480</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(70, 255, 76);</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelChoseSum">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Valitse nostettava summa </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnArrow7">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>220</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn50">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>440</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>20€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>40€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_50">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>50€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_100">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>100€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_back">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>300</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Takaisin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_difSum">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Muu summa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>481</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>btn4</zorder>
  <zorder>btn40</zorder>
  <zorder>btnFetchTransactions</zorder>
  <zorder>btnDifSum</zorder>
  <zorder>labelChosenSum</zorder>
  <zorder>btn6</zorder>
  <zorder>btnStop</zorder>
  <zorder>btn100</zorder>
  <zorder>btnClear</zorder>
  <zorder>btn2</zorder>
  <zorder>btn1</zorder>
  <zorder>btn20</zorder>
  <zorder>btn5</zorder>
  <zorder>btn8</zorder>
  <zorder>btn7</zorder>
  <zorder>btnArrow8</zorder>
  <zorder>btn0</zorder>
  <zorder>btnOk</zorder>
  <zorder>btn3</zorder>
  <zorder>labelChoseSum</zorder>
  <zorder>btnArrow7</zorder>
  <zorder>btn50</zorder>
  <zorder>btn9</zorder>
  <zorder>label_20</zorder>
  <zorder>label_40</zorder>
  <zorder>label_50</zorder>
  <zorder>label_100</zorder>
  <zorder>label_back</zorder>
  <zorder>label_difSum</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
